Doc: Improve documentation for new users

strokesws opened the pull request:
Hello lovely people!

First and foremost, thank you for the incredible work on this project. I've been following it closely and am very impressed. I wanted to find a way to contribute, and I noticed a common theme in the community (on YouTube, Reddit, and in comments) of new users feeling a bit lost on how to get started. I also ran into some confusion myself, as it's been a while since I've worked on Linux.

To help with this, I've rewritten some sections of the project's wiki to be clearer and more accessible for newcomers. I've created a pull request with these changes in a private repository for you to review: https://github.com/strokesws/lsfg-vk-wiki-fork/pull/1. I hope you find the changes helpful!

In addition to the wiki, I would like to suggest a few changes to the README.md file to better serve as a high-level guide. The goal is to make the entry point for new users more direct by pointing them to the specific, detailed wiki pages for installation and configuration.

I've done my best to preserve the original tone and your "PancakeTAS touch" in the documentation. The credit section and the technical documentation about the development process were not altered at all, as those are important parts of the project's history and credit.

Please let me know what you think of the proposed changes. I'm open to any feedback or suggestions you might have.

Thank you again for all your work!

Commit:
refactor: review readme file explanation

PancakeTAS commented:
I appreciate the changes! I haven't been able to look at them fully yet but while I'm here I might as well say a few things:
1. I believe the various distribution-specific build guides should be merged into one document. The only real changes are the package managers and for some distributions the install location.
2. I saw that you got rid of the `$` in most code blocks, I'm not sure I like that...
3. In the Debian installation section you mention `dpkg -i` but as far as I know this won't install remote dependencies
4. I updated the quirks page inbetween you writing the document and adding your changes, so the diff seems incorrect.

I'll take a closer look at the changes when I find the time, thank you!

strokesws commented:
You're welcome! I'm glad that I can help!

>     1. I believe the various distribution-specific build guides should be merged into one document. The only real changes are the package managers and for some distributions the install location.
I updated the PR, they should all be unified in one file. Hopefully I didn't miss anything important.

>     2. I saw that you got rid of the `$` in most code blocks, I'm not sure I like that...
The idea was to make life easier for lazy people like me 🥺, if you click on the "Copy" button the `$` was included and you had to manually remove it. Something similar with the script to retrieve the process names, the terminal wouldn't allow you to run multi-line scripts (at least with fish), that's why I added a one-liner version. Let me know what you prefer.

>     3. In the Debian installation section you mention `dpkg -i` but as far as I know this won't install remote dependencies
You're probably correct, fixed.

>     4. I updated the quirks page inbetween you writing the document and adding your changes, so the diff seems incorrect.
Updated!

Commit:
refactor: replace the build guide link

PancakeTAS committed:
more wiki changes
